intera_examples/scripts/send_random_trajectory.py

```diff
@@ -0,0 +1,160 @@
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import rospy
+import motion_msgs.msg
+import argparse
+from intera_motion_interface import (
+    MotionTrajectory,
+    MotionWaypoint,
+    MotionWaypointOptions,
+    RandomWalk
+)
+from motion_msgs.msg import TrajectoryOptions
+from intera_core_msgs.msg import JointLimits
+from intera_interface import (
+    Limb,
+    JointLimits
+)
+
+def main():
+    """
+    Send a random-walk trajectory, starting from the current pose of the robot.
+    Can be used to send both joint and cartesian trajectories.
+
+    Call using:
+    $ rosrun intera_examples send_random_trajectory.py  [arguments: see below]
+
+    -n 5 -t JOINT -s 0.5
+    --> Send a trandom joint trajectory with 5 waypoints, using a speed ratio
+        of 0.5 for all waypoints. Use default random walk settings.
+
+    -d 0.1 -b 0.2
+    --> Send a random trajectory with default trajectory settings. Use a
+        maximum step distance of 0.1*(upper joint limit - lower joint limit)
+        and avoid the upper and lower joint limits by a boundary of
+        0.2*(upper joint limit - lower joint limit).
+
+    -o ~/Desktop/fileName.bag
+    --> Save the trajectory message to a bag file
+
+    -p
+    --> Prints the trajectory to terminal before sending
+
+    """
+    arg_fmt = argparse.RawDescriptionHelpFormatter
+    parser = argparse.ArgumentParser(formatter_class=arg_fmt,
+                                     description=main.__doc__)
+    parser.add_argument(
+        "-n", "--waypoint_count", type=int, default=5,
+        help="number of waypoints to include in the trajectory")
+    parser.add_argument(
+        "-d", "--stepDistance", type=float, default=0.05,
+        help="normalized random walk step distance")
+    parser.add_argument(
+        "-b", "--boundaryPadding", type=float, default=0.1,
+        help="normalized padding to apply to joint limits")
+    parser.add_argument(
+        "-t", "--trajType", type=str, default='JOINT',
+        choices=['JOINT', 'CARTESIAN'],
+        help="trajectory interpolation type")
+    parser.add_argument(
+        "-s",  "--speed_ratio", type=float, default=0.5,
+        help="A value between 0.0 (slow) and 1.0 (fast)")
+    parser.add_argument(
+        "-a",  "--accel_ratio", type=float, default=0.5,
+        help="A value between 0.0 (slow) and 1.0 (fast)")
+    parser.add_argument(
+        "-o", "--output_file",
+        help="Save the trajectory task to a bag file")
+    parser.add_argument(
+        "--output_file_type", default="yaml",
+        choices=["csv", "yaml"], help="Select the save file type")
+    parser.add_argument(
+        "-p", "--print_trajectory", action='store_true', default=False,
+        help="print the trajectory after loading")
+    parser.add_argument(
+        "--do_not_send", action='store_true', default=False,
```

Comment:
I am tempted to default this to `True`... I am slightly concerned about the potential for an unwitting user to invoke this script and collide the robot with the environment, or controller box.